src/logic/safe/store/reducer/safe.ts

```diff
@@ -88,7 +88,7 @@ export default handleActions<AppReduxState['safes'], Payloads>(
       return shouldUpdate
         ? state.updateIn(
             ['safes', safeAddress],
-            makeSafe({ name: safe?.name || 'LOADED SAFE', address: safeAddress }),
+            makeSafe({ name: safe?.name || 'Gnosis Safe', address: safeAddress }),
```

Comment:
Should we now filter this name as we were doing with 'LOADED_SAFE' ?

Reply:
Yes 👍